src/components/WalletSelectionComponent.jsx

```diff
@@ -135,6 +135,12 @@ class WalletSelectionComponent extends Component {
       )
     }
 
+    if (walletType === 'ledger' && !ledgerNanoS.connected) {
```

Comment:
Refer to the layout in https://projects.invisionapp.com/d/#/console/16692554/346151385/preview for error display

left: cross icon with red background
right: 'Please make sure your device is connected' (copied from myetherwallet)